clr-k8s-examples/9-multi-network/multus-sriov-ds.yaml

```diff
@@ -87,6 +87,20 @@ data:
         user: multus
     current-context: multus-context
     EOF
+  install-multus-conf.sh: |
```

Comment:
@krsna1729 is this cleaned up? What will happen the next time this script runs? Will it not see 00-multus.conf at /host/etc/cni/net.d

Reply:
@mcastelino  Ooh thats right. Good catch.